.eslintrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
   rules: {
     // allow reassigning param
     'no-param-reassign': [2, { props: false }],
+    'linebreak-style': 0,
```

Comment:
For next time, I would recommend committing unrelated things separately.

Reply:
Thanks, will do next time.